README.md

```diff
@@ -73,6 +72,13 @@ We're using tailwind utility classes so please try and use them as much as possi
 
 In progress, though can be [seen on Figma](https://www.figma.com/file/VUh53RdncjZ7NuFYj0RGB9/CadHub?node-id=0%3A1)
 
-<img src="https://raw.githubusercontent.com/Irev-Dev/repo-images/main/images/Part%20Page(1).jpg">
+## Integrations
+The OpenSCAD and CadQuery integrations work by leveraging each of their cli tools in a docker image. It's currently deployed to AWS and can be found [here](https://github.com/Irev-Dev/cadhub/tree/main/app/api/src/docker).
 
-<img src="https://raw.githubusercontent.com/Irev-Dev/repo-images/main/images/User%20Page%20Edit.jpg">
+## Docs
+Docs are hosted at [learn.cadhub.xyz](http://learn.cadhub.xyz/). It includes a OpenSCAD tutorial at this point, and more is coming. The docs can be found in this repo at [docs](https://github.com/Irev-Dev/cadhub/tree/main/docs)
+
+
+## Who is CadHub
+
+[Kurt](https://github.com/Irev-Dev) and [Frank](https://github.com/franknoirot) make up the Core-team and [Jeremy](https://github.com/jmwright) is a major contributor. Plus a number smaller contributors.
```

Comment:
It's a bit arbitrary. Are you happy with this @franknoirot, @jmwright?

Reply:
I appreciate the mention, but I'm not sure I'd be considered a major contributor. The only feature I helped with significantly was the CadQuery integration.

Reply:
Sounds like a major contribution to me. I say we leave it there unless you would rather I remove it?

Reply:
If you would like to leave it, that's fine with me.